src/panels/RetinaCapturePanel.ts

```diff
@@ -0,0 +1,169 @@
+import { Uri, window, workspace } from "vscode";
+import { BasePanel, PanelDataProvider } from "./BasePanel";
+import { InitialState, ToVsCodeMsgDef } from "../webview-contract/webviewDefinitions/retinaCapture";
+import { MessageHandler } from "../webview-contract/messaging";
+import { TelemetryDefinition } from "../webview-contract/webviewTypes";
+import { withOptionalTempFile } from "../commands/utils/tempfile";
+import { KubectlVersion, invokeKubectlCommand } from "../commands/utils/kubectl";
+import * as k8s from "vscode-kubernetes-tools-api";
+import { failed } from "../commands/utils/errorable";
+import * as vscode from "vscode";
+import { longRunning } from "../commands/utils/host";
+import { relative } from "path";
+import { platform } from "os";
+import open from 'open';
+
+
+export class RetinaCapturePanel extends BasePanel<"retinaCapture"> {
+    constructor(extensionUri: Uri) {
+        super(extensionUri, "retinaCapture", {
+            startCaptureResponse: "",
+            getAllNodesResponse: [],
+        });
+    }
+}
+
+export class RetinaCaptureProvider implements PanelDataProvider<"retinaCapture"> {
+    constructor(
+        readonly kubectl: k8s.APIAvailable<k8s.KubectlV1>,
+        readonly kubeConfigFilePath: string,
+        readonly kubectlVersion: KubectlVersion,
+        readonly clusterName: string,
+        readonly retinaOutput: string,
+        readonly allNodeOutput: string[],
+        readonly captureFolderName: string
+    ) { }
+
+    getTitle(): string {
+        return `Retina Distributed Capture on ${this.clusterName}`;
+    }
+
+    getTelemetryDefinition(): TelemetryDefinition<"retinaCapture"> {
+        return {
+            getAllNodes: false,
+            runRetinaCapture: true,
+            retinaCaptureResult: true,
```

Comment:
Out of these, only one seems to be implemented, so we probably shouldn't be collecting telemetry for either of the other two (which should be removed?)